_guides/005-operations.md

```diff
@@ -20,6 +20,10 @@ sections:
     title: Extend storage capacity
     description: Extend physical capacity
     module: operations/extend-storage
+  -
+    title: External Key/Value 
```

Comment:
I think this should say 'Using an external etcd cluster'

Reply:
done